src/overlays/ovl_i4/fox_fo.c

```diff
@@ -1,29 +1,1513 @@
+                actorPtr[1].obj.rot.z = -90.0f;
+            }
+            if (gTeamShields[3] > 0) {
+                func_i4_801890EC(&gActors[2], 2);
+                actorPtr[2].obj.pos.x = player->pos.x;
+                actorPtr[2].obj.pos.y = player->pos.y + 100.0f + 400.0f;
+                actorPtr[2].obj.pos.z = player->unk_138 - 250.0f;
+            }
+            func_i4_801890EC(gActors, 3);
+
+            actorPtr[0].obj.pos.z = player->pos.z + 400.0f;
+            actorPtr[0].vel.z = 0.0f;
+            actorPtr[0].info.bonus = 1;
+            gCsFrameCount = 0;
+            player->unk_1D0 = 21;
+            player->unk_234 = 1;
+
+            for (i = 0; i < 9; i++) {
+                D_80177A48[i] = 0.0f;
+            }
+
+            D_80177A48[5] = 94.0f;
+            D_801779A0 = 0.0f;
+            D_801779B8 = 30.0f;
+            D_801779C0 = 0.0f;
+
+        case 21:
+            if (gCsFrameCount < 30) {
+                break;
+            }
+
+            if (D_80178340 != 0) {
+                D_80178358 = 0;
+                D_8017835C = 1;
+            }
+
+            if (gCsFrameCount == 140) {
+                player->unk_0D0 = 30.0f;
+                actorPtr[3].vel.z = 30.0f;
+                actorPtr[1].vel.z = 30.0f;
+                actorPtr[2].vel.z = 30.0f;
+                actorPtr[0].vel.z = 30.0f;
+            }
+            if (gCsFrameCount > 140) {
+                Math_SmoothStepToF(&gActors[3].obj.rot.z, 0.0f, 0.03f, 1.2f, 0.0001f);
+                Math_SmoothStepToF(&gActors[3].obj.pos.x, -100.0f, 0.03f, 1000.0f, 0.0001f);
+                Math_SmoothStepToF(&gActors[3].obj.pos.y, 40.0f, 0.03f, 1000.0f, 0.0001f);
+            }
+            if (gCsFrameCount > 180) {
+                Math_SmoothStepToF(&gActors[1].obj.rot.z, 0.0f, 0.03f, 1.2f, 0.0001f);
+                Math_SmoothStepToF(&gActors[1].obj.pos.x, 100.0f, 0.03f, 1000.0f, 0.0001f);
+                Math_SmoothStepToF(&gActors[1].obj.pos.y, 40.0f, 0.03f, 1000.0f, 0.0001f);
+            }
+            if (gCsFrameCount > 220) {
+                Math_SmoothStepToF(&gActors[2].obj.pos.y, 90.0f, 0.03f, 1000.0f, 0.0001f);
+            }
+            if (gCsFrameCount > 340) {
+                Math_SmoothStepToF(&player->pos.y, -10.0f, 0.03f, 1000.0f, 0.0001f);
+            }
+            if (gCsFrameCount > 530) {
+                Radio_PlayMessage(gMsg_ID_20010, RCID_FOX);
+                player->pos.y = -10.0f;
+                actorPtr[3].obj.rot.z = 0.0f;
+                actorPtr[3].obj.pos.x = -100.0f;
+                actorPtr[3].obj.pos.y = 40.0f;
+                actorPtr[1].obj.rot.z = 0.0f;
+                actorPtr[1].obj.pos.x = 100.0f;
+                actorPtr[1].obj.pos.y = 40.0f;
+                actorPtr[2].obj.pos.y = 90.0f;
+                player->unk_1D0 = 22;
+            }
+
+            switch (gCsFrameCount) {
+                case 0x8C:
+                    func_8001D444(0U, 0x26U, 0U, 0xFFU);
```

Comment:
```suggestion
                    func_8001D444(0, 0x26, 0, 0xFF);
```
I think you can get rid of most of the Us you have here